src/Take.Blip.Builder/FlowSessionManager.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System.Threading;
+using System.Threading.Tasks;
+
+namespace Take.Blip.Builder
+{
+    /// <summary>
+    /// Defines a service for managing the current flow session for a given user.
+    /// </summary>
+    public class FlowSessionManager : IFlowSessionManager
+    {
+        private const string CURRENT_FLOW_SESSION_KEY = "currentFlowSession";
+
+        /// <summary>
+        /// Gets the current flow session for the user in the flow.
+        /// </summary>
```

Comment:
You don't need to add xml documentation in the implementation files, just in the interface files.

Reply:
currently not adding xml documentation in the implementation file, generates warnings